runtime/compiler/env/j9method.cpp

```diff
@@ -7769,7 +7769,7 @@ TR_J9ByteCodeIlGenerator::runFEMacro(TR::SymbolReference *symRef)
             if (strcmp(sourceType, targetType))
                {
                char methodName[30], methodSignature[50];
-               if (sourceType[0] == 'L' && isExplicit)
+               if (sourceType[0] == 'L' && isExplicit) //not sure if Q needs to be added here
```

Comment:
If I'm reading the code correctly, if a `sourceSig` with `Q` is mapped to `sourceType` of `Ljava/lang/Object;` above, then this line won't have to deal with `Q`.